homeassistant/components/sense/__init__.py

```diff
@@ -111,9 +113,22 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     except SENSE_TIMEOUT_EXCEPTIONS:
         raise ConfigEntryNotReady
 
+    trends_coordinator = DataUpdateCoordinator(
+        hass,
+        _LOGGER,
+        name=f"Sense Trends {email}",
+        update_method=gateway.update_trend_data,
+        update_interval=timedelta(seconds=300),
+    )
+
+    # async_request_refresh instead of async_refresh
+    # as this can take longer than 60s
+    await trends_coordinator.async_request_refresh()
```

Comment:
This will still trigger the fetch and block until it is finished.

Reply:
Oh nevermind, default debouncer of update coordinator is not set to immediate

Reply:
Uuh, I was wrong. It is set to immediate by default. So this is doing a full refresh.

Reply:
Thanks.  Switched to `async_call_later` unless you can suggest something better?